hooks/secondary_pre_publish_tk-3dsmaxplus.py

```diff
@@ -82,16 +82,16 @@ def execute(self, tasks, work_template, progress_cb, user_data, **kwargs):
             item = task["item"]
             output = task["output"]
             errors = []
+            app = self.parent
 
             # report progress:
             progress_cb(0, "Validating", task)
 
-            # pre-publish item here, e.g.
-            #if output["name"] == "foo":
-            #    ...
-            #else:
-            # don't know how to publish this output types!
-            errors.append("Don't know how to publish this item!")
+            if output["name"] == "alembic_cache":
```

Comment:
Looks like the items are added as `geometry` type, but the rest of the hooks are looking for `alembic_cache` type. Am I missing something? If not, then this is broken in Maya.